src/app/(dashboard)/meetings/[meetingId]/page.tsx

```
@@ -0,0 +1,11 @@
+import React from "react";
+
+interface Props {
+  params: Promise<{ meetingId: string }>;
+}
+
+const MeetingPage = ({}: Props) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)